### DISCUSSION TITLE
a way to stop the sync

### DISCUSSION THREAD

**Discussion by sojan1** [Mimetis/Dotmim.Sync#1119] [Q&A] [answered]
Is there a way to stop the Sync while synching is running?

Scenerio:  A user started synching but suddenly became mindful about something and wanted to revert back to the old.

Thanks & Regards

**Comment by mcash95** [comment, marked as answer]
I'll link the docs section for interceptors for you. I think that may be the solution to your dilemma. SynchronizeAsync() also may be able to take a cancellationtoken (havent looked into this yet). Best of luck!

https://dotmimsync.readthedocs.io/Interceptors.html

**Comment by sojan1** [comment]
Thanks I will have a look at it.
Cheers
